apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -59,7 +62,10 @@ const Ticket = new GQLListSchema('Ticket', {
             knexOptions: { isNotNullable: true }, // Required relationship only!
             kmigratorOptions: { null: false, on_delete: 'models.PROTECT' },
         },
-
+        order: {
```

Comment:
Does this field overrides sorting order only at display phase in UI?

Reply:
No, this field is needed for server-side sorting